src/BlueprintCommand.php

```diff
@@ -50,7 +50,7 @@ public function handle()
         }
 
         $contents = $this->files->get($file);
-        $blueprint = resolve(Blueprint::class);
+        $blueprint = $this->resolve(Blueprint::class);
```

Comment:
If this is all that is needed to support Lumen, please simply change the `resolve` helper to the `app` helper and I'll merge it.

Reply:
Jason, I realized that Lumen not support FormRequest, so the pull is incomplete.